### PR TITLE
espnet2 zeroth_korean recipe changing feats_type from fbank_pitch to raw.

### PR DESCRIPTION
**Pull request by hchung12**
The previous recipe used fbank_pitch features and measured performance using valid.acc.best model, and WER was 2.3%.

This recipe changes feats_type from fbank_pitch to raw and some hyper parameters are tuned.
In this recipe, WER is 2.4% for valid.acc.best, but 2.0% for valid.acc.ave model.

### CODE REVIEW

**Comment by kamo-naoyuki** [comment]
Thanks!
Could you append your result at the top of README.md instead of replacing it? I mean, I'd like to keep the previous one.

**Comment by hchung12** [comment]
I keep the previous result using fbank_pitch features. However, the previous result was reevaluated with valid.acc.ave instead of valid.acc.best because performance is better.

**Comment by kamo-naoyuki** [comment]
Okay, thanks!

File: egs2/TEMPLATE/asr1/path.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 MAIN_ROOT=$PWD/../../..
 KALDI_ROOT=$MAIN_ROOT/tools/kaldi
 
-export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$PATH
+export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$KALDI_ROOT/tools/sctk/bin:$PATH
```
Comment (kamo-naoyuki) [inline egs2/TEMPLATE/asr1/path.sh:4]:
```suggestion
export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$PATH
```

Please not edit this.

File: egs2/TEMPLATE/asr1/path.sh [hunk 2]
```diff
@@ -19,4 +19,4 @@ export PYTHONIOENCODING=UTF-8
 export NCCL_SOCKET_IFNAME="^lo,docker,virbr,vmnet,vboxnet"
 
 # NOTE(kamo): Source at the last to overwrite the setting
-. local/path.sh
+#. local/path.sh
```
Comment (kamo-naoyuki) [inline egs2/TEMPLATE/asr1/path.sh:22]:
```suggestion
. local/path.sh
```
Please not edit this.

**Pull request merged**